packages/finance/src/liability/liquidation.rs

```diff
@@ -559,17 +585,68 @@ mod tests {
         );
     }
 
+    #[test]
+    fn min_liquidation() {
+        let max_ltv = Percent::from_permille(751);
+        let spec = liability_with_max(max_ltv);
+
+        assert_eq!(
+            check_liability::<Nls>(
+                &spec,
+                878.into(),
+                752.into(),
+                0.into(),
+                800.into(),
```

Comment:
[all occurrences within this test] select a corner value, not like here `800` which is far from the threshold. The idea is to prove that the result No is returned at that corner value while a value Liquidate is returned at the value + 1